pushconfig.sh

```diff
@@ -38,10 +39,12 @@ Available options :
     -V                   Be verbose.
     -d                   Use this script in dry run: no change will be made.
     -h                   Display this help.
+    -F                   Set JSON 'path' to exclude from configuration files. Multiple 'path' are specified using a semi-colon separated list using the jq format.
+    -o                   Override extension of .json configuration file. If set, any file named confXXX.json-<override-extension> will be used instead of confXXX.json.
```

Comment:
I do not understand the usecase for the "-o", what will it do ?
If it is used for suffixing some files; would not that be better to put the suffix before the extension (confXXX_suffix.json) to keep IDE understanding the file types ?

Reply:
L'idée du suffixe après le JSON était de ne pas casser le fonctionnement actuel. Le '_' est déjà utilisé dans la l'algorithme mais le use case n'est pas précisé. 

Sais-tu dans quels cas le '_' est utilisé en prod ?

Reply:
Tu parles de ça ?
```
        filename=$(basename -- "${configfile}") # Name of the file, without repertory
        output_name="${filename%.*}"            # remove the .json file extension
        service="${output_name%%_*}"            # service name is the 1st part of the filename, before "_"
```
On dirait vu le code et le commentaire que ça sers à permettre de nommer ton fichier un peu comme tu veux (tu ne peux pas avoir N fichiers pour un service par contre vu comment c'est codé). Je ne l'ai jamais vu utilisé.

Mais je ne comprends pas vraiment quel est l'use case pour ta modification, c'est pour avoir des sortes de variations de configurations sans devoir avoir un dossier entier ?

Reply:
c'est ça

Reply:
D'ac. A part les restes (la doc du `-F` ça me semble bon alors) :+1:

Reply:
Nouvelle version poussée sur Github, avec prise en compte des remarques, dont une meilleure intégration avec l'IDE.